guides/source/configuring.md

```diff
@@ -1205,6 +1205,27 @@ end
 config.active_record.migration_strategy = CustomMigrationStrategy
 ```
 
+#### `config.active_record.schema_versions_formatter`
+
+Controls the formatter class used by schema dumper to format versions information. Custom class can be provided
+to change the default behavior:
+
+```ruby
+class CustomSchemaVersionsFormatter
```

Comment:
I've just tried this snippet and it's missing 
```ruby
   private attr_reader :connection
   def initialize(connection)
        @connection = connection
   end
```

Reply:
cc @fatkodima

Reply:
oops, https://github.com/rails/rails/pull/54054